src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -385,6 +388,24 @@ export default class PatientViewMutationTable extends MutationTable<
                 this.getSamples().length > 1
             );
         };
+
+        // generate namespace columns
+        const namespaceColumns = createNamespaceColumns(
+            this.props.namespaceColumns
+        );
+        _.forIn(
+            namespaceColumns,
+            (column: MutationTableColumn, columnName: string) => {
+                if (
+                    this.props.columns &&
+                    !this.props.columns.includes(columnName)
+                ) {
+                    this.props.columns.push(columnName);
```

Comment:
looks like we are mutating a prop here? that's antipattern.  lets discuss.

Reply:
Fixed.